flink-kubernetes/src/main/java/org/apache/flink/kubernetes/configuration/KubernetesConfigOptions.java

```diff
@@ -221,6 +221,21 @@
 	/** Defines the configuration key of that external resource in Kubernetes. This is used as a suffix in an actual config. */
 	public static final String EXTERNAL_RESOURCE_KUBERNETES_CONFIG_KEY_SUFFIX = "kubernetes.config-key";
 
+	public static final ConfigOption<Map<String, String>> KUBERNETES_SECRETS =
+		key("kubernetes.secrets")
+			.mapType()
+			.noDefaultValue()
+			.withDescription("The user-specified secrets that will be mounted into Flink container. The value should be in " +
+				"the form of foo:/opt/secrets-foo,bar:/opt/secrets-bar.");
```

Comment:
I think it would be good to add a bit more description (might even want to add this description to `native_kubernetes.md`) for how to configure the secrets. At least we should add a link to the K8s documentation https://kubernetes.io/docs/concepts/configuration/secret/.

Reply:
add some description in [native_kubernetes.md](https://github.com/apache/flink/pull/12899/files#diff-8ac5ff0acb9a268297bd157249c52acdR214)